### DISCUSSION TITLE
Error: request to https://api.vercel.com/v2/user failed, reason: certificate has expired

### DISCUSSION THREAD

**Discussion by mageaustralia** [vercel/community#2246] [Bug Report]
If I run the following:

bash# npx vercel link
Vercel CLI 29.0.2
? Set up “~/Development/blog-nextjs-sanity”? [Y/n] y
Error: request to https://api.vercel.com/v2/user failed, reason: certificate has expired

If I run an openssl check, I get this, yet visiting in my browser, everything works fine.

openssl s_client -host https://api.vercel.com -port 443
CONNECTED(00000003)
depth=0 CN = zeit.co
verify error:num=18:self signed certificate
verify return:1
depth=0 CN = zeit.co
verify error:num=10:certificate has expired
....

    Start Time: 1682892679
    Timeout   : 300 (sec)
    Verify return code: 10 (certificate has expired)

**Comment by mageaustralia** [comment]
api.verce.com resolves to 76.76.21.112

ping api.vercel.com
PING api.vercel.com (76.76.21.112): 56 data bytes
64 bytes from 76.76.21.112: icmp_seq=0 ttl=120 time=14.759 ms
64 bytes from 76.76.21.112: icmp_seq=1 ttl=120 time=6.563 ms
64 bytes from 76.76.21.112: icmp_seq=2 ttl=120 time=6.840 ms

And if I access the IP address directly in the browser, I get this:
<img width="418" alt="zeit co_screenshot" src="https://user-images.githubusercontent.com/1046615/235378644-0b4d8e9f-967b-4e01-925b-e1884110ebf3.png">

**Comment by amyegan** [comment]
Hi @trabulium! Are you still having this problem? I wasn't able to reproduce the error myself

**Reply by mageaustralia** [reply]
Hi, yes, I still have the issue. 

host -t A api.vercel.com
api.vercel.com has address 76.76.21.112 

wget api.vercel.com 
--2023-05-02 05:53:56--  http://api.vercel.com/
Resolving api.vercel.com (api.vercel.com)... 76.76.21.112
Connecting to api.vercel.com (api.vercel.com)|76.76.21.112|:80... connected.
HTTP request sent, awaiting response... 308 Permanent Redirect
Location: https://api.vercel.com/ [following]
--2023-05-02 05:53:56--  https://api.vercel.com/
Connecting to api.vercel.com (api.vercel.com)|76.76.21.112|:443... connected.
ERROR: cannot verify api.vercel.com's certificate, issued by ‘CN=R3,O=Let's Encrypt,C=US’:
  Issued certificate has expired.
To connect to api.vercel.com insecurely, use `--no-check-certificate'.

If I try with --no-check-certificate


wget api.vercel.com --no-check-certificate
--2023-05-02 05:54:57--  http://api.vercel.com/
Resolving api.vercel.com (api.vercel.com)... 76.76.21.112
Connecting to api.vercel.com (api.vercel.com)|76.76.21.112|:80... connected.
HTTP request sent, awaiting response... 308 Permanent Redirect
Location: https://api.vercel.com/ [following]
--2023-05-02 05:54:58--  https://api.vercel.com/
Connecting to api.vercel.com (api.vercel.com)|76.76.21.112|:443... connected.
WARNING: cannot verify api.vercel.com's certificate, issued by ‘CN=R3,O=Let's Encrypt,C=US’:
  Issued certificate has expired.
HTTP request sent, awaiting response... 308 Permanent Redirect
Location: https://vercel.com/docs/api [following]
--2023-05-02 05:54:58--  https://vercel.com/docs/api
Resolving vercel.com (vercel.com)... 76.76.21.164, 76.76.21.241
Connecting to vercel.com (vercel.com)|76.76.21.164|:443... connected.
WARNING: cannot verify vercel.com's certificate, issued by ‘CN=R3,O=Let's Encrypt,C=US’:
  Issued certificate has expired.
HTTP request sent, awaiting response... 308 Permanent Redirect
Location: /docs/rest-api [following]
--2023-05-02 05:54:59--  https://vercel.com/docs/rest-api
Reusing existing connection to vercel.com:443.
HTTP request sent, awaiting response... 200 OK
Length: 319235 (312K) [text/html]
Saving to: ‘index.html’

If you throw this in your browser, you should get the bad SSL. https://76.76.21.112/ - btw, I'm in Melbourne, Australia, in case it's a specific edge server that has the outdated certificate. I also tried using a VPN (London, San Francisco and Manchester) and got the same error each time. Finally, it's worth noting that if I just try http://76.76.21.112/, it redirects correctly to my dashboard, I specifically need to hit https://76.76.21.112/ to get the SSL error.
